test/test_atoms.jl

```diff
@@ -10,7 +10,7 @@ using Test
 
 # Do not use `using LinearAlgebra` to check symbols are re-exported by Convex.
 import LinearAlgebra
-
+import Clarabel
```

Comment:
I'd rather _not_ have solvers in these tests. They should all just be logic tests. Solvers should go in `problem_depot`.

I'll make some changes to this PR.